api/src/main/resources/swagger3codegen/aouTracedMethodCall.mustache

```diff
@@ -16,7 +16,7 @@
 
         try {
             // Create and execute the API request.
-            {{#returnType}}ApiResponse<{{{returnType}}}> {{localVariablePrefix}}resp = {{/returnType}}{{operationId}}WithHttpInfo({{#allParams}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/allParams}});
+            {{#returnType}}ApiResponse<{{{returnType}}}> {{localVariablePrefix}}resp = {{/returnType}}{{operationId}}WithHttpInfo({{#parameters}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/parameters}});
```

Comment:
I am completely failing to understand why this change works. The current codegen works properly on the 1.0.2 template. AFAICT the swagger codegen version used for swagger3 was not upgraded in the plugin. This is unsettling, but I don't have further theories on this.